buildenv/jenkins/JenkinsfileBase

```diff
@@ -193,12 +193,18 @@ def buildTest() {
 			try {
 				//get pre-staged jars from test.getDependency build before test compilation
 				copyArtifacts fingerprintArtifacts: true, projectName: "test.getDependency", selector: lastSuccessful(), target: 'openjdk-tests/TestConfig/lib'
-				
-				//get pre-staged test jars from systemtest.getDependency build before system test compilation
-				copyArtifacts fingerprintArtifacts: true, projectName: "systemtest.getDependency", selector: lastSuccessful(), target: 'openjdk-tests/TestConfig/lib'
 			} catch (Exception e) {
 				echo 'Cannot run copyArtifacts from test.getDependency. Skipping copyArtifacts...'
 			}
+			
+			try { 
+				if (env.BUILD_LIST == 'systemtest') { 	
+					//get pre-staged test jars from systemtest.getDependency build before system test compilation
+					copyArtifacts fingerprintArtifacts: true, projectName: "systemtest.getDependency", selector: lastSuccessful(), target: 'openjdk-tests'
```

Comment:
Should we download into `openjdk-tests/systemtest_prereqs`?

Reply:
Sorry, did I miss somehting? The code here means we download it to `openjdk-tests`.

Reply:
We are copying `systemtest_prereqs` folder from systemtest.getDependency build and placing it under openjdk-tests. https://github.com/AdoptOpenJDK/openjdk-tests/blob/1114a6936698ee0cd4adb86beb95b866837ebd6d/buildenv/jenkins/getDependency#L26

Reply:
ah, thanks.